validator/enforce_labels.rego

```diff
@@ -91,6 +92,35 @@ get_labels(asset, non_standard_types) = resource_labels {
 	resource_labels := lib.get_default(resource, "resourceLabels", {})
 }
 
+# get_labels for Spanner Instances. API format.
+# This method matches data generated by Terraform Validator and other
+# tools that use the API format to represent resources.
+get_labels(asset, non_standard_types) = resource_labels {
+	# check if we have a non-standard type
+	asset.asset_type == non_standard_types[_]
```

Comment:
Note, this line is redundant/unnecessary.

Reply:
Removed